biosimulations/apps/dispatch-api/src/simulation-run/simulation-run.controller.ts

```diff
@@ -123,6 +129,20 @@ export class SimulationRunController {
 
     const run = await this.service.createRun(parsedRun, file);
 
+    // console.log('start to Dispatch with filename: ', file.originalname)
+    this.messageClient.send(MQDispatch.SIM_DISPATCH_START,
+      {
+        simulationId: run.id,
+        omexFileName: file.originalname,
+        simulator: run.simulator,
+        simulatorVersion: run.simulatorVersion
+      })
+      .subscribe(dat => {
+        // TODO: Make MQ work without this subscription
```

Comment:
If you make this into emit rather than send it will not need a subscription. However, We do want this to be a message because we want the dispatch service to return an acknowledgment that the run was accepted.

Reply:
Yes, I checked with emit, but the problem persists the same.